scrc/dataset_creation/judgment_dataset_creator.py

```diff
@@ -52,5 +52,5 @@ def plot_custom(self, df, split_folder, folder):
 if __name__ == '__main__':
     config = get_config()
 
-    judgment_dataset_creator = JudgmentDatasetCreator(config)
-    judgment_dataset_creator.create_dataset(sub_datasets=True, kaggle=False, save_reports=True)
+    judgment_dataset_creator = JudgmentDatasetCreator(config, debug=True)
+    judgment_dataset_creator.create_multiple_datasets(concatenate=True, overview=True)
```

Comment:
I thought that you did not want to concatenate in the judgment_dataset_creator?

Reply:
Do you mean because of the "True"-state?
It was just the last thing I tried when testing.